express/blocks/link-list/link-list.css

```diff
@@ -3,6 +3,10 @@ main .section > div.link-list-wrapper {
   padding-right: 0;
 }
 
+main .section:has( .link-list) {
```

Comment:
can you add paddings & margins to the block, but not to the section if possible?